app/src/main/java/com/benefit/model/Match.java

```diff
@@ -53,4 +63,8 @@ public int getId() {
     public void setClosed(boolean closed) {
         isClosed = closed;
     }
+
+    public List<String> getUsersId() {
+        return usersId;
```

Comment:
instead of adding 2 more fields to the class & the constructors, why not returning here Arrays.asList(sellerId,buyerId) ?

Reply:
I have to store it as a list and the getter must return the same list